toolbox/util/githubClient.go

```diff
@@ -370,15 +372,20 @@ func (g GithubClient) GetTagCommitSHA(repo, tag string) (string, error) {
 
 // GetCommitCreationTime gets the time when the commit identified by sha is created
 func (g GithubClient) GetCommitCreationTime(repo, sha string) (time.Time, error) {
```

Comment:
All the methods on GithubClient can be changed to *
```
func (g *GithubClient) GetCommitCreationTime(repo, sha string) (time.Time, error) {
```

Reply:
Done. Thanks!